### DISCUSSION TITLE
v7.0-(1403-2-12)

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/MahsaNG#38] [Announcements]
[mahsa_net channel](https://t.me/mahsa_net)

customized xray-core for iran
unblock wireguard
fakehost to unblock non-tls config
ability to chain warp before any config
fix ping issue of fragmented config
handle mux switch on server
smart parameters update on fetch
routing rules are now optional
better handling of custom json config
fix quic transport inconsistency
UI & speed improvement

Known Issue:
warp chain may not work with config that use CF domain as address

<hr /><em>This discussion was created from the release <a href='https://github.com/GFW-knocker/MahsaNG/releases/tag/v7.0-(1403-2-12)'>v7.0-(1403-2-12)</a>.</em>

**Comment by sayhigh369963** [comment]
111
